data/ships.txt

```diff
@@ -437,16 +437,18 @@ ship "Behemoth"
 		"X3200 Ion Steering"
 		"Scram Drive"
 		
-	engine -34 154
-	engine 34 154
-	gun -27 -83
-	gun 27 -83
-	turret -34 -64 "Heavy Laser Turret"
-	turret 34 -64 "Heavy Anti-Missile Turret"
-	turret -58 11 "Heavy Laser Turret"
-	turret 58 11 "Heavy Laser Turret"
-	turret -24 78 "Heavy Anti-Missile Turret"
-	turret 24 78 "Heavy Laser Turret"
+	gun 47.5 -57.5
+	gun -47.5 -57.5
+	engine 34 143 0.8
+	engine -34 143 0.8
+	engine 56 148.5 0.6
+	engine -56 148.5 0.6
+	turret 47.5 33.5 "Heavy Laser Turret"
+	turret -47.5 33.5 "Heavy Laser Turret"
+	turret 22 66.5 "Heavy Laser Turret"
+	turret -22 66.5 "Heavy Anti-Missile Turret"
+	turret 31 -50 "Heavy Anti-Missile Turret"
+	turret -31 -50 "Heavy Laser Turret"
```

Comment:
Should be engine, gun, turret, not gun, engine, turret. The Heavy Shuttle hardpoints were also shuffled around.
Also, since hardpoint coordinates are saved in the save file, realize that drastically moving hardpoints will make the ships of current saves look weird.